zenoh/tests/routing.rs

```diff
@@ -27,7 +27,7 @@ use zenoh_core::ztimeout;
 use zenoh_protocol::core::{WhatAmI, WhatAmIMatcher};
 use zenoh_result::bail;
 
-const TIMEOUT: Duration = Duration::from_secs(10);
+const TIMEOUT: Duration = Duration::from_secs(60);
```

Comment:
Why do we need to increase this ?

Reply:
Because waiting until the task terminate gracefully requires some time, with 10 seconds I observe a lot of tracked tasks still alive by the end of the test.

Reply:
fixed back to 10 seconds. Termination seems to work faster once I switched  to Tokio executor.